Build/Projects/MonoGame.Framework.definition

```diff
@@ -292,7 +292,7 @@
     <Compile Include="DisplayOrientation.cs" />
     <Compile Include="DrawableGameComponent.cs" />
     <Compile Include="FrameworkDispatcher.cs">
-      <Platforms>_FrameworkDispatcherProvided</Platforms>
+      <Services>_FrameworkDispatcherProvided</Services>
```

Comment:
This is causing the Linux build to fail.

We should probably remove this and just make `FrameworkDispatcher.cs` supported on all platforms.

I don't see any reason now that it should not be.